modules/end-user-guide/partials/proc_allowing-users-to-define-workspace-creation-strategy.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// creating-a-workspace-from-a-remote-devfile
+
+[id="allowing-users-to-define-workspace-creation-strategy_{context}"]
+= Allowing users to define workspace creation strategy
+
+
+This section describes how to control the strategy of creating a {prod-short} workspace using a factory URL. {prod-short} currently supports the following options: 
+
+* `perclick`:  `perclick` is the default creation strategy, which creates a new workspace each time the same factory URL is accepted.
+
+* `peruser`: `peruser` creation strategy will re-use an existing workspace for second and any other subsequent acceptance of the same factory URL.
```

Comment:
```suggestion
* `peruser`: For a specific combination of user and factory URL, reuse the existing workspace.
```
Avoid repetition. Explain key information (per user) even if it is implied by something else (factory URL?). Put the condition first. Simplify.